test/setup_mongo_wendy.js

```diff
@@ -0,0 +1,14 @@
+var DB = require('mongodb').Db;
+var Server = require('mongodb').Server;
+var db = new DB('test', new Server(process.env.MONGODB_HOST || "localhost", 27017));
+db.open()
+  .then(d => d.admin())
+  .then(a => a.addUser('admin','admin'))
+  .then(x => {
+    console.log(x);
+    process.exit(0);
+  })
+  .catch(e => {
```

Comment:
If you didn't explicitly catch here, would it print the error message anyway, and exit non-zero?